docs/tutorial/index.md

```diff
@@ -105,135 +145,151 @@ You can also watch juju logs with the [`juju debug-log`](https://juju.is/docs/ju
+           |          |          |                 |         |         |            |           | monitoring=c/operator             +
+           |          |          |                 |         |         |            |           | charmed_databases_owner=c/operator
+
 ```
 
-The output should be something like this:
+In order to run queries, we can enter the `psql` interactive terminal by running the following command, again typing the password when requested:
+
+```{terminal}
+:input: sudo psql --host=10.1.110.80 --username=operator --password postgres
+:user: ubuntu
+:host: juju-1c143d-0
+
+Password:
+```
+
+After submitting the password, you'll enter an interactive terminal like this:
 
 ```text
-psql (14.10 (Ubuntu 14.10-0ubuntu0.22.04.1))
+psql (16.9 (Ubuntu 16.9-0ubuntu0.24.04.1))
 Type "help" for help.
 
-postgres=## 
+postgres=# 
 ```
 
-Now you are successfully logged in the interactive terminal. Here it is possible to execute commands to PostgreSQL directly using PostgreSQL SQL Queries. For example, to show which version of PostgreSQL is installed, run the following command:
+Now you are successfully logged in the `psql` interactive terminal. Here it is possible to execute commands to PostgreSQL directly using PostgreSQL SQL Queries. For example, to show which version of PostgreSQL is installed, run the following command:
 
 ```text
-postgres=## SELECT version();
-                                                             version
----------------------------------------------------------------------------------------------------------------------------------
- PostgreSQL 14.10 (Ubuntu 14.10-0ubuntu0.22.04.1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0, 64-bit
+postgres=# SELECT version();
+                                                               version                                                                
+--------------------------------------------------------------------------------------------------------------------------------------
+ PostgreSQL 16.9 (Ubuntu 16.9-0ubuntu0.24.04.1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0, 64-bit
 (1 row)
 ```
 
-We can see that PostgreSQL version 14.10 is installed. From this prompt, to print the list of available databases, we can simply run this command:
+We can see that PostgreSQL version 16.9 is installed. From this prompt, to print the list of available databases, we can simply run this command:
 
 ```text
-postgres=## \l
+postgres=# \l
 ```
 
 The output should be the same as the one obtained before with `psql`, but this time we did not need to specify any parameters since we are already connected to the PostgreSQL application.
 
 To create and connect to a new sample database, we can run the following commands:
 
 ```text
-postgres=## CREATE DATABASE mynewdatabase;
-postgres=## \c mynewdatabase
+postgres=# CREATE DATABASE mynewdatabase;
```

Comment:
We should mentione here something like: we are creating database here manually for edication purpouse only, while the proper way has been described below using data-integrator or direct relation with db client app.

As we are not recomending used to create Databases manually (objects ownership, etc).